indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -195,4 +195,27 @@
             $(this).closest('tr').toggleClass('selected', this.checked);
         }).trigger('change');
     };
+
+    global.showThemeSettingsForm = function showThemeSettingsForm(formUrl, lastFieldId) {
```

Comment:
We usually pass an `options` dict to these functions instead of having separate arguments (see some other global functions for examples)